ngirimana/murikahomeltd
# #173658569 resolve some state errors

> #### What does this PR do?
- persist state using local storage
> #### Description of Task to be completed
- fetch token and expiration date
- calculate expiration time
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **bg-resolve-some-state-errors-173658569**  branch by running ```git checkout bg-resolve-some-state-errors-173658569``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the login link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173658569](https://www.pivotaltracker.com/story/show/173658569)
> #### Screenshots (if appropriate)
* N/A
